### ISSUE TITLE
Design Time Issue With Return Type of Task

### ISSUE THREAD

**Issue by lbrecheisen** [max-ieremenko/ServiceModel.Grpc#79]
## Describe the bug
Task only return type causes build error with source code generation.

## To Reproduce

```c#
[ServiceContract]
public interface IExampleService
{
    [OperationContract]
    Task Example(int number, CancellationToken token);
}
```

```c#
internal partial class ExampleService : IExampleService
{
    public async Task Example(int number, CancellationToken token)
    {
        await Task.CompletedTask;
    }
}
```

```c#
[ExportGrpcService(typeof(ExampleService), GenerateAspNetExtensions = true)]
internal static partial class Services { }
```

```c#
[ImportGrpcService(typeof(IExampleService)]
internal static partial class Services { }
```

## Expected behavior:
Using Task  as a return time should not cause error with design time code.

### Environment
OS: Windows 11
.NET 6

**Comment by max-ieremenko** [comment]
Thank you for the bug report,

Unfortunately i cannot reproduce neither with ServiceModel.Grpc 1.4.4 nor with latest 1.4.5.

Here is the attached solution with the code you provided [test#79.zip](https://github.com/max-ieremenko/ServiceModel.Grpc/files/8357383/test.79.zip)
Could you double check, maybe I missed something.

**Comment by lbrecheisen** [comment]
Thanks for creating a solution and looking into this. I should have been a bit more specific with the reproduction code. 

Looks like I had a namespace in a project containing `Message`. That name appeared to be conflicting with a type during source generation. I've since changed it to `Messages` and that fixed the issue for me.

I've modified your solution to reproduce that error. [test.79.zip](https://github.com/max-ieremenko/ServiceModel.Grpc/files/8364182/test.79.zip)
